docs/resources/sql_permissions.md

```diff
@@ -31,6 +31,8 @@ resource "databricks_sql_permissions" "foo_table" {
 }
 ```
 
+It is recommended to define all permissions for a securable in a single resource, otherwise Terraform cannot guarantee config drift prevention.
```

Comment:
```suggestion
It is required to define all permissions for a securable in a single resource, otherwise Terraform cannot guarantee config drift prevention.
```